app/helpers/template_helpers.py

```diff
@@ -15,7 +15,7 @@
 
 
 @lru_cache(maxsize=None)
-def get_page_header_context(language, theme):
+def get_page_header_context(language: str, theme: str) -> dict:
```

Comment:
```suggestion
def get_page_header_context(language: str, theme: str) -> Mapping:
```

dict should be avoided in favour of `typing.Mapping` or `typing.MutableMapping` where possible

Reply:
Thanks, added. I think it got resolved in previous Andrew comment.